components/responsive-wrapper/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress Dependencies
+ */
+import { cloneElement } from 'element';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+function ResponsiveWrapper( { naturalWidth, naturalHeight, children } ) {
+	if ( ! children || Array.isArray( children ) ) {
```

Comment:
I think we don't want to test children as an array, but rather `Children.count > 1` 

https://facebook.github.io/react/docs/react-api.html#react.children.count

Or even `Children.only` if we don't mind being very noisy (thrown errors) with incorrect usage:

https://facebook.github.io/react/docs/react-api.html#react.children.only